packages/node_modules/@cerebral/http/src/HttpProviderError.js

```diff
@@ -1,11 +1,11 @@
 export default class HttpProviderError extends Error {
-  constructor(status, headers, body, message = null, isAborted = false) {
+  constructor(status, headers, result, message = null, isAborted = false) {
     super()
     this.name = 'HttpProviderError'
     this.message = message
     this.status = status
     this.headers = headers
-    this.body = body
+    this.result = result
```

Comment:
Yeah, I can see why you would want that @gaspard , but the `result` is not the `responseText`. If the error response was JSON, it will be parsed, meaning it is rather a result of that. But `result` is generic. Maybe move back to body?